presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -312,6 +312,7 @@ public final class SystemSessionProperties
     public static final String REWRITE_LEFT_JOIN_ARRAY_CONTAINS_TO_EQUI_JOIN = "rewrite_left_join_array_contains_to_equi_join";
     public static final String REWRITE_LEFT_JOIN_NULL_FILTER_TO_SEMI_JOIN = "rewrite_left_join_null_filter_to_semi_join";
     public static final String USE_BROADCAST_WHEN_BUILDSIZE_SMALL_PROBESIDE_UNKNOWN = "use_broadcast_when_buildsize_small_probeside_unknown";
+    public static final String PREFER_BROADCAST_JOIN_OVER_PARTITIONED_JOIN = "prefer_broadcast_join_over_partitioned_join";
```

Comment:
This will change the default behavior of join planning, and fail query when input estimation is not accurate, i.e. the input size is inaccurately predicated to be small but not. I would like to keep conservative here.

Reply:
I mean rename the session property to reflect the actual action